src/component.js

```diff
@@ -99,6 +100,7 @@ function createSubmitHandler (hostedFieldsInstance, orderIdFunction) : Function
 
         return {
           liabilityShifted,
+          liabilityShift:       payload.liability_shift,
```

Comment:
are we going to return both liabilityShifted and liabilityShift for all clients?

Reply:
We've planned to deprecate `liabilityShifted` for all partners and direct merchants. Going forward for new integration, only `liabilityShift` would be encouraged to be used.